.gitignore

```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+.env
```

Comment:
missing newline at the end of the file
```suggestion
.env

```

Reply:
the newline is actually there. try to refresh